README.md

```diff
@@ -120,3 +135,295 @@ This will return the following runtime images with the following names: `action-
 ```
 ./gradlew :tests:test
 ```
+
+---
+
+## Using Docker <a name="using_docker"></a>
+
+### Pre-requisites
+- [Docker](https://www.docker.com/)
+- [curl](https://curl.se/), [wget](https://www.gnu.org/software/wget/), or [Postman](https://www.postman.com/)
+
+
+0. Choose/create a folder of your liking and make sure docker daemon is running
+
+1. Clone this repo:
+```
+git clone https://github.com/apache/openwhisk-runtime-nodejs
+cd openwhisk-runtime-nodejs
+```
+
+1.1 Choose a NodeJS version. All build files reside inside `core/nodejsActionBase`. If you take a look into `core/nodejsActionBase/Dockerfile` you’ll see a line that looks like:
+```
+FROM node:lts-stretch
+```
+This will use the latest NodeJS version. But we want to be more specific. Now if you look into each of the Dockerfile’s of `core/nodejs14Action`, `core/nodejs12Action`, `core/nodejs10Action`, you’ll notice different nodeJS versions. Let’s go ahead with the 14 version. We'll be creating a `build` folder that will contain all the necessary files for us to build our NodeJS container. All you have to do is copy paste the commands below. This will copy the NodeJS application as well as our target Dockerfile with the NodeJS version 14.
+
+```
+mkdir build
+cp -r core/nodejsActionBase/* build
+cp core/nodejs14Action/Dockerfile build
+```
+
+If you follow the instructions at beginning of this tutorial [here](#build_gradle) that uses Gradle, you'll notice that Gradle takes care of this copying for us internally. Here since we just want to use docker and not worry about anything else we make create our own little build folder.
```

Comment:
Do not refer to this page as a "tutorial"; instead add a subsection "Resetting the build"